### PR TITLE
wrong name for the min-overlap option

### PR DESCRIPTION
**Pull request by marina-manrique**

### CODE REVIEW

**Comment by marina-manrique** [comment]
@laughedelic I think I fixed this. It is already published.

**Comment by laughedelic** [comment]
So should the string in the end be `--min-overlap`? here it will be converted to `--min_overlap` if I understand right

File: src/test/scala/CommandGeneration.scala [hunk 1]
```diff
@@ -26,8 +26,8 @@ class CommandGeneration extends FunSuite {
         (new File("reads2.fastq")).getCanonicalPath.toString,
         "--output-prefix", "sample1",
         "--output-directory", (new File("/tmp/out")).getCanonicalPath.toString,
-        "--minOverlap", "10",
-        "--maxOverlap", "65",
+        "--min-overlap", "10",
```
Comment (marina-manrique) [inline src/test/scala/CommandGeneration.scala:29]: @laughedelic apparently it won't, this test passes

**Comment by laughedelic** [comment]
Ah, ok, I see now `replace("_", "-")` :+1:

**Pull request merged**